docs/development/extensions-core/s3.md

```diff
@@ -38,7 +38,7 @@ The AWS SDK requires that the target region be specified. Two ways of doing this
 As an example, to set the region to 'us-east-1' through system properties:
 
 - Add `-Daws.region=us-east-1` to the jvm.config file for all Druid services.
-- Add `-Daws.region=us-east-1` to `druid.indexer.runner.javaOpts` in middleManager/runtime.properties so that the property will be passed to Peon (worker) processes.
+- Add `-Daws.region=us-east-1` to `druid.indexer.runner.javaOpts` in [middleManager/runtime.properties](../../configuration/index.md#middlemanager-configuration) so that the property will be passed to Peon (worker) processes.
```

Comment:
I think it would be more correct to just refer to it as middle manager configuration instead of 'middleManager/runtime.properties', since you don't necessarily have to use those files.